data-canary/scripts/creaturescripts/login.lua

```diff
@@ -97,7 +97,16 @@ function login.onLogin(player)
 	-- Set Client XP Gain Rate --
 	if configManager.getBoolean(configKeys.XP_DISPLAY_MODE) then
 		local baseRate = player:getFinalBaseRateExperience()
-		player:setBaseXpGain(baseRate * 100)
+		baseRate = baseRate * 100
+		if configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED) then
+			local vipBonusExp = configManager.getNumber(configKeys.VIP_BONUS_EXP)
+			if (vipBonusExp > 0 and player:isVip()) then
```

Comment:
Lua does not use parentheses "ifs" of simple conditions.

```suggestion
			if vipBonusExp > 0 and player:isVip() then
```